### PR TITLE
Accept keys with different formatting

### PR DESCRIPTION
**Pull request by danpalmer**

### CODE REVIEW

**Comment by danpalmer** [comment]
Not quite ready yet, please do not merge.

**Comment by danpalmer** [comment]
@graingert Good to go, please review.

Note: in YAML raw `0x...` values come out as numbers in JS, so I've got a type assertion first to provide a useful error message, but then still handling the case of a YAML string starting `"0x..."` in case someone uses quotes.

File: index.mjs [hunk 1]
```diff
@@ -39,7 +39,12 @@ const userNamesFromKey = _.flow(
 
 async function validateFile(filename) {
   const content = await fs.readFile(filename);
-  const { name, fingerprint } = yaml.safeLoad(content);
+  const { name, fingerprint: rawFingerprint } = yaml.safeLoad(content);
+  assert(
+    typeof rawFingerprint == 'string',
```
Comment (graingert) [inline index.mjs]: ===

**Review by graingert — CHANGES_REQUESTED** [review]

File: index.mjs (hunk 1 above)
Comment (danpalmer) [reply index.mjs]: a8e0e12

**Review by graingert — APPROVED** [review]

**Comment by danpalmer** [comment]
@graingert re-review please

**Pull request merged**